Core/gesture/__init__.py

```diff
@@ -63,19 +62,51 @@ def click():
         pyautogui.click()
 
 
+def double_click():
+    print('double-click')
+    if sys.platform == 'windows':
+        print('click on windows')
+    else:
+        pyautogui.click()
+        time.sleep(.1)
+        pyautogui.click()
+
+
+
+def reset_mouse(x=3840/2,y=2160/2):
+    size = pyautogui.size()
+    print(size)
+    print(x)
+    print(y)
+    pyautogui.moveTo(x,y)
+
+
+
+
 class HandDetection():
     def __init__(self):
         # Camera Params
         cap_device = 0
         cap_width = 1280
         cap_height = 720
 
+        # Screen params
+        self.screen_x = 3840
```

Comment:
see if you can auto get the screen height and width instead of just hard coding the value

Reply:
we should probably configure this from a settings file rather than auto-detecting